frontend/src/components/charts/NetworkChart.tsx

```diff
@@ -85,9 +95,10 @@ export const NetworkChart = ({ networkData }: NetworkChartProps) => {
             type: "graph",
```

Comment:
Lets apply the following styles in the series attribute so that the node do not block the edges when it becomes large

itemStyle: {
                opacity: 0.8,
              },

Reply:
Already applied in line 107.